plotly/tools.py

```diff
@@ -145,7 +146,8 @@ def reset_credentials_file():
 def set_config_file(plotly_domain=None,
                     plotly_streaming_domain=None,
                     plotly_api_domain=None,
-                    plotly_ssl_verification=None):
+                    plotly_ssl_verification=None,
+                    plot_options=None):
```

Comment:
@theengineear - agreed: hiding `plot_options` here is confusing. agreed, let's de-nest it and add `world_readable` as an explicit argument (and in the future, `secret_link` :) ). the only benefit to having `plot_options` is that it implicitly matches the kwargs of `plot`, but I don't think anyone besides us will get that.

Just be clear:

- `sign_in` - accepts de-nested arguments of `config`, `credentials`
- `plot`, `iplot`- accepts de-nested `world_readable`, `secret_link`
- `set_credentials` - accepts `credentials`
- `set_config` - accepts `config` and `world_readable`, `secret_link`
- `session.set_plot_options` - (or w/e its called) - accepts `plot_options` including  `world_readable`, `secret_link`